VERSION

```diff
@@ -1 +1 @@
-9.7.11
+9.7.12
```

Comment:
could also be 9.8.0 if you want

Reply:
Yeah it definitely makes more sense to bump the minor version, mb